### PR TITLE
add rqlite database

### PR DESCRIPTION
**Pull request by NerdyShawn**
Thank you for wanting to submit a Pull Request to the Civo Kubernetes Marketplace repository!

If your pull request is to submit a new application to the marketplace, please answer the following questions:

* [x] Have you followed the guidelines in our Contributing document in setting up the application?
* [X] Have you checked that the application your submission is proposing to add does not yet exist on the Marketplace, or as a pull request to be processed?
* [x] Have you tested your application on the Civo managed Kubernetes service? Please include a screenshot.
* [X] Can you confirm that the software you are submitting licensed for use in a publicly-available Kubernetes marketplace?
* [x] Have you included the specific version of the application to be installed?

If your pull request concerns an existing Marketplace application, please make sure you have:

* [X] Notified the Maintainer of the application in this pull request so that they are aware of your proposal.
* [X] Outlined the changes you are proposing, and the reasons these are required (e.g. stability, compatibility with new versions of Kubernetes implementations, etc).
* [x] Tested that the application works with the proposed updates applied (including a screenshot).
* [x] Updated the version number of the app if that has changed.

### CODE REVIEW

**Comment by NerdyShawn** [comment]
Validated Install and uninstall. This brings up a single replica with 10GB storage, for now sticking pretty close to vanilla helm chart values. In the future perhaps more SSL, auto backup & auto restore to s3 object store.
6641503B-92BD-4B3C-A7B1-E29ECE98B198

File: rqlite/post_install.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+## Rqlite - The lightweight, distributed relational database built on SQLite.
```
Comment (otoolep) [inline rqlite/post_install.md:1]: I always spell it with a lowercase 'r', FWIW.  As in "rqlite - the lightweight, distributed relational database built on SQLite."

Comment (NerdyShawn) [reply rqlite/post_install.md]: Thanks Philip for the extra 👀 I updated this and validated lowercase everywhere else too. While you're here I just grabbed the logo.png from the site not sure if you have a different image you want in. It should work but if we want to fill it out the space more we can max up to 512 x 512. More details here on the image sizing if you think we should tweak it at all.
- https://github.com/civo/kubernetes-marketplace#writing

Comment (otoolep) [reply rqlite/post_install.md]: No problem using the logo, go for it.

File: rqlite/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+name: rqlite
+title: rqlite
+version: 8.16.3
+maintainer: "shawn@nebarilabs.com"
+description: "The lightweight, distributed relational database built on SQLite."
+url: https://rqlite.io
+category: database
+dependencies:
```
Comment (saiyam1814) [inline rqlite/manifest.yaml:9]: If no dependencies then we can remove this

**Review by saiyam1814 — APPROVED** [review]
LGTM

**Pull request merged**